mission_control/navigator_launch/launch/cameras.launch

```diff
@@ -0,0 +1,80 @@
+<launch>
+  <!-- This is the default pointgrey launch file with our id's and our names  -->
+
+  <arg name="front_camera_serial" default="15223362" />
+  <arg name="front_camera_calibrated" default="0" />
+  <arg name="down_camera_serial" default="10282860" />
+  <arg name="down_camera_calibrated" default="0" />
```

Comment:
Go for it, do whatever you have to do to get it up and running for Saturday. Just make sure that PTGrey launch works.